hardware/Netatmo.cpp

```diff
@@ -1872,10 +2202,19 @@ bool CNetatmo::ParseDashboard(const Json::Value& root, const int DevIdx, const i
 		// Check when dashboard data was last updated
 		if (!root["time_utc"].empty())
 			tNetatmoLastUpdate = root["time_utc"].asUInt();
-		//Debug(DEBUG_HARDWARE, "Module [%s] last update = %s", name.c_str(), ctime(&tNetatmoLastUpdate));
+//		else
+//		{
+//			Log(LOG_STATUS, "No time stamp in received data; using current time");
+//			tNetatmoLastUpdate = tNow;
+//		}
+		//Debug(DEBUG_HARDWARE, "Module [%s] last update = %s (%d)", name.c_str(), ctime(&tNetatmoLastUpdate), tNetatmoLastUpdate);
+		Log(LOG_STATUS, "Module [%s] last update = %s (%d)", name.c_str(), ctime(&tNetatmoLastUpdate), tNetatmoLastUpdate);
+
 		// check if Netatmo data was updated in the past NETAMO_POLL_INTERVALL (+1 min for sync time lags)... if not means sensors failed to send to cloud
 		int Interval = NETAMO_POLL_INTERVALL + 60;
 		//Debug(DEBUG_HARDWARE, "Module [%s] Interval = %d", name.c_str(), Interval);
+		Log(LOG_STATUS, "Module [%s] Interval = %d", name.c_str(), Interval);
```

Comment:
I think this should be a DEBUG_HARDWARE ?

Reply:
Sorry, but I (miss-)use LOG_STATUS for logging/debugging, because when I enable debugging, I get a lot of debug messages from other devices that will distract my attention of the important message to understand what is happening.

I will remove them later (when I get some results from other beta testers with devices I don't own).

Reply:
if you want you can also add

#ifdef _DEBUG
... do something
#endif

Then this will only be displayed when you build in debug mode (which you do while developing)

Reply:
That would not help, because after merging to the beta release, it will be built without the _DEBUG value defined. I will need this information to check the logs from (BETA-) users that experience issues or have devices I don't own. 

By the way, I don't have a proper debug environment (with a decent symbolic) setup yet, because the first trials to do so failed, and I didn't have the time to figure that out. So no useful need for debug mode so far... Sorry!

Reply:
Maybe try to install/use visual studio? Community version is free and great!

Reply:
Thanks for the tip: I used this for several years to development in C# on a Windows system. But can this be used to debug on a Raspberry Pi, running Linux? In need to figure this out, but too late for this project, I guess.